pulp_container/tests/functional/api/test_push_content.py

```diff
@@ -188,3 +189,44 @@ def test_push_to_existing_namespace(self):
         # cleanup, namespace removal also removes related distributions
         namespace = self.namespace_api.list(name="team").results[0]
         self.addCleanup(self.namespace_api.delete, namespace.pulp_href)
+
+    def test_private_repository(self):
```

Comment:
can you add one more test?
Test that you can create a private distribution and push to it.
Test that the same user_creator can pull, but user_helpless cannot.
Test the user_reader can pull form private repo.
Test that the other user_helpless  can pull after marking it non-private.

Reply:
Apart from "Test the user_reader can pull form private repo." that sounds exactly like the test you are looking at. I can add this action to it, if you like ?!

Reply:
yes i meant adding only the  Test the user_reader can pull form private repo to add, I basically added it to the list of what you are already have.